KubeArmor/enforcer/appArmorTemplate.go

```diff
@@ -80,7 +80,7 @@ profile {{.Name}} flags=(attach_disconnected,mediate_deleted) {
 	## == PRE END == ##
 
 	## == POLICY START == ##
-{{range $value, $data := .FilePaths}}{{$suffix := ""}}{{if and $data.Dir $data.Recursive}}{{$suffix = "{,**}"}}{{else if $data.Dir}}{{$suffix = "{,*}"}}{{end}}{{if $data.Deny}}{{if and $data.ReadOnly $data.OwnerOnly}}
+{{range $value, $data := .FilePaths}}{{$suffix := ""}}{{if and $data.Dir $data.Recursive}}{{$suffix = "{,**,.**}"}}{{else if $data.Dir}}{{$suffix = "{,*}"}}{{end}}{{if $data.Deny}}{{if and $data.ReadOnly $data.OwnerOnly}}
```

Comment:
Is this problem limited to recursive directories? Does it work fine when recursive is not set?